src/CADL.Extension/Emitter.Csharp/src/options.ts

```diff
@@ -60,6 +62,11 @@ export const NetEmitterOptionsSchema: JSONSchemaType<NetEmitterOptions> = {
             type: "array",
             nullable: true,
             items: { type: "string" }
+        },
+        logLevel: {
+            type: "string",
+            enum: ["error", "warn", "info", "debug", "verbose"],
```

Comment:
Should the values of this enum come from `logger.ts`?

Reply:
Yes